woocommerce/woocommerce
# Update/default payments gateways recommendations

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR updates the default payment gateway recommendation priority to match the per-country recommendations and order in [this spreadsheet](https://docs.google.com/spreadsheets/d/1cb4M2oa9C6H6I287JSM_oc_s_IP8VzipnTb1TMI8FbY/edit#gid=1417956487).

Note: I followed the lead of https://github.com/woocommerce/woocommerce/pull/37442.


### How to test the changes in this Pull Request:



Review the changes and make sure the order aligns with [this spreadsheet](https://docs.google.com/spreadsheets/d/1cb4M2oa9C6H6I287JSM_oc_s_IP8VzipnTb1TMI8FbY/edit#gid=1417956487)

1. Go to OBW and select US as store country
2. Complete OBW with WCPay installed
3. Go to `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com`
4. Untick `Display suggestions within WooCommerce`
5. Go to WooCommerce > Home > Payment task
6. Set up the WCPay
7. Go to WooCommerce > Home > AdditionalPayments task
8. Observe that payment gateways are shown in the following order: 

- PayPal
- AmazonPay
- Affirm
- Afterpay
- Klarna
- Zip Co

<img src="https://user-images.githubusercontent.com/4344253/227878256-4da3ec2a-b8d2-410c-8e75-6a526163ba5e.png"  width="500px" />

9. Change store country to `Germany`
10. Go to  AdditionalPayments task
11. Observe that payment gateways are shown in the following order: 

- PayPal
- Klarna Payments
- AmazonPay



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
